backend/lcfs/db/models/notification/NotificationType.py

```diff
@@ -18,11 +20,10 @@ class NotificationType(BaseModel, Auditable):
     __table_args__ = {"comment": "Represents a Notification type"}
 
     notification_type_id = Column(Integer, primary_key=True, autoincrement=True)
-    name = Column(
-        Enum(NotificationTypeEnum, name="notification_type_enum", create_type=True),
-        nullable=False,
-    )
+    name = Column(String(255), nullable=False)
```

Comment:
Most needed change, thanks for it.

Reply:
Thanks! Glad this simplifies things for future additions.